### PR TITLE
Add view basic styles

### PR DESCRIPTION
**Pull request by tiagomarin**
### :heavy_check_mark: Implement the design from the sneak peek wireframes.
- Everything functions. (Note: Ignore login logic, you would be using Devise in coming milestone)
  - All Users page wireframe:
  - Single User page wireframe:
  - All Posts by a given User page wireframe:
  - Single Post page wireframe:    

### :heavy_check_mark: Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
- **Hint:** Data was added to the database through the `seed.rb` file

### :speech_balloon: Feel free to shoot me a message in the pull request if you have any questions.

Thank you for your time!

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+  <div class="flex flex-col gap-3 w-full p-4 border border-black border-solid">
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  </div>
+  <ul class="flex flex-col gap-3 w-full">
+    <p class="only:block hidden">There are no posts yet!</p>
+    <% @user.last_3_posts.each do |post| %>
+      <%= link_to user_post_path(@user, post.id) do %>
+        <%= render partial:'shared/post_highlight', locals: {user: @user, post: post} %>
+      <% end %>
```
Comment (danmainah) [inline app/views/users/show.html.erb]: - [x] @tiagomarin great work in your views, but you forgot something small, you didn't display the number of comments and likes in the post as it's in the design check the comparison below:

Design Given          |  Your design     
:-------------------------:|:-------------------------:
image |  image

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
     <% end %>
-  <% end %>
-</ul>
-<button>Show all posts</button>
\ No newline at end of file
+  </ul>
+  <button>Show all posts</button>
+</div>
```
Comment (danmainah) [inline app/views/users/show.html.erb]: - [x] Kindly ensure this button is active and when clicked takes the user to the post index page,
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @tiagomarin , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/views/users/show.html.erb (hunk 1 above)
Comment (tiagomarin) [reply app/views/users/show.html.erb]: ### :heavy_check_mark: Changes applied

Thanks for pointing that out! :smile:

File: app/views/users/show.html.erb (hunk 2 above)
Comment (tiagomarin) [reply app/views/users/show.html.erb]: ### :heavy_check_mark: Changes applied

Thanks for pointing that out! :smile:

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @tiagomarin 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tiagomarin** [comment]
Thank you for reviewing my code! 🙏

**Review by KayLemba — COMMENTED** [review]
[# STATUS: APPROVED :green_circle: :clap: 

Hello  @tiagomarin       , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: aaa
Congratulations! 🎉 

HIGHLIGHTS:

- [x]  Your repo is professional 
- [x] Descriptive P.R








 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]

**Pull request merged**